### PR TITLE
feat(forms): add forms

### PR DESCRIPTION
**Pull request by olros**
## Proposed changes
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

Issue number: closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Pull request title follows [conventional commits](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) (`type(scope): description`)
- [ ] Build (`make PR`) was run locally without failures

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by olros — COMMENTED** [review]
Trenger detail-tilbakemelding når et form slettes

**Review by Her0elt — APPROVED** [review]

File: app/content/serializers/event.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Meta:
             "sign_off_deadline",
             "registration_priorities",
             "evaluate_link",
+            "forms",
```
[the remark this answers is not in the document]
Comment (Eirsteir) [reply app/content/serializers/event.py:38]: Én id til survey og én til evaluering?

[the remark this answers is not in the document]
Comment (olros) [reply app/content/serializers/event.py:38]: Jepp

**Review by Her0elt — APPROVED** [review]
looks good :writing_hand:

**Pull request merged**